lighthouse-core/audits/predictive-perf.js

```diff
@@ -71,6 +72,34 @@ class PredictivePerf extends Audit {
     return scriptUrls;
   }
 
+  /**
+   * @param {!Node} dependencyGraph
+   * @return {!Object}
+   */
+  static computeOptions(dependencyGraph) {
+    const records = [];
+    dependencyGraph.traverse(node => {
+      if (node.type === Node.TYPES.NETWORK) records.push(node.record);
+    });
+
+    const rttSummaries = Array.from(NetworkAnalyzer.estimateRTTByOrigin(records).entries());
```

Comment:
later on you use this pattern, which i think is a lot more readable:

```js
      for (const [origin, summary] of rttByOrigin.entries()) {
        rttByOrigin.set(origin, summary.min);
      }
```
does take more lines, but worth it IMO

Reply:
yeah cleaned this whole thing up